### DISCUSSION TITLE
Is it possible to omit the inspect-brk arg?

### DISCUSSION THREAD

**Discussion by jjohnson1994** [mfussenegger/nvim-dap#752] [Q&A] [answered]
We're using a framework called [Bit](https://bit.dev/) to bundle components in our app. To run tests in Bit you use the command `bit test`, and to run those tests in debug mode and attach a debugger you use `bit test --debug`.

Adding any additional args (e.g. when nvim-dap appends `--inspect-brk`) results in an error: "Unknown arguments: inspect-brk, inspectBrk".

Is it possible to omit the `--inspect-brk` param for some configs?

**Comment by jjohnson1994** [comment]
some more context, with the same launch.json
VSCode runs:
`/Users/jjohnson1994/bin/bit test --debug`
nvim-dap runs:
`/Users/jjohnson1994/bin/bit test --debug --inspect-brk=49647`

**Comment by mfussenegger** [comment, marked as answer]
nvim-dap doesn't add the `--inspect-brk` flag. It must be coming from elsewhere. 

Looks like some of the extensions are setting it? https://cs.github.com/?scopeName=All+repos&scope=&q=inspect-brk+language%3Alua

**Reply by jjohnson1994** [reply]
thanks, guess it's coming from the adaptor then, I'll have a look there 👍
